cli/waiter/subcommands/kill.py

```diff
@@ -1,36 +1,44 @@
 import logging
+import requests
 from urllib.parse import urljoin
 
 from tabulate import tabulate
 
 from waiter import http_util, terminal
 from waiter.format import format_last_request_time, format_status
-from waiter.querying import get_service, print_no_data, query_service, query_services
+from waiter.querying import print_no_data, query_service, query_services
 from waiter.util import guard_no_cluster, str2bool, response_message, print_error, wait_until, check_positive
 
 
 def kill_service_on_cluster(cluster, service_id, timeout_seconds):
     """Kills the service with the given service id in the given cluster."""
-
-    def service_is_killed():
-        return get_service(cluster, service_id)['status'] == 'Inactive'
-
     cluster_name = cluster['name']
+    http_util.set_retries(0)
     try:
         print(f'Killing service {terminal.bold(service_id)} in {terminal.bold(cluster_name)}...')
-        resp = http_util.delete(cluster, f'/apps/{service_id}')
+        params = {'timeout': timeout_seconds * 1000}
+        read_timeout = 5 if timeout_seconds == 0 else timeout_seconds
```

Comment:
Why default to 5 if user chose 0 seconds? Avoid passing it (i.e. read_timeout=None) and rely on default global timeout values in that situation?

Reply:
I saw this default being used in ping.py and wasn't sure
https://github.com/twosigma/waiter/blob/02470066f5037bec706b2258adaa046f30e9c6a2/cli/waiter/subcommands/ping.py#L16